src/act.other.cpp

```diff
@@ -699,6 +713,9 @@ ACMD(do_shapechange) {
     mob->desc = ch->desc;
     ch->desc = nullptr;
     ch->forward = mob;
+
```

Comment:
We should install `clang-format` or something for you to auto-clean up these empty lines.